src/ghe_api.py

```diff
@@ -14,7 +14,7 @@
 
 def get_access_token(code):
 	if DEV_MODE:
-		return "some_token"
+		return code
```

Comment:
I probably do not understand this change. The code is different from the access token. The code is used to get access token in the real world. I know it won't matter but I think it will be nice to draw the distinction between code and access token.

Reply:
Yeah the OAuth authorization code is pretty different from an Access Token, and is usually just used once to get a token, as far as I remember.

Reply:
So the “code” passed in will be the user name. Then by returning the code, a user name is returned as the access token. Later in the next section, the access token passed in will be the user name. This is minimal change I made to make it work only in dev but not in production. It will probably be better to add extra logic in the function that calls this function instead.

Reply:
We should probably change the argument name then - it looks confusing.

Reply:
I agree that this will work but in the developer workflow the `code` variable holds `username` but in production, the code is something different. It is not good practice to use a variable to hold something it does not represent. So re-naming does not fix this issue only exacerbates it.